doc/todo/RELEASE

```diff
@@ -9,6 +9,59 @@ Following shell expansions are used in this document:
  ~elektra-ftp .. where ftp files go to
  ~elektra-doc .. where doc flies go to
 
+----------------------------------------------------------------------
+
+This describes what to do for the automated release:
+
+= Tasks before Release =
+
+== Updates ==
+
+- Update doc/COMPILE.md to reflect actually tested setups
+- Write release notes
+
+== Increment Version Number ==
+
+- CMakeLists.txt
+- ~e/scripts/dev/build-debian-package
+- Change VERSION variable in build-server
+- Commit and push the changes
+
+== Check ==
+
+- Everything blue on https://build.libelektra.org/
+- Trigger release pipeline with dry run
+- Inspect the logs, artifacts and the website from the pipeline
+  - [ ] error logs (*.error) must be empty
+  - [ ] check in the (test-*) folder if all tests passed
+  - [ ] check if version is correct in the version file
+  - [ ] 
+
+== Preperation ==
+- Finalize release notes (hash sums etc are already known)
+- cd ~e && cp doc/todo/NEWS.md doc/news/_preparation_next_release.md
+
+== Publish ==
+- Trigger release pipeline without dry run
+- (Possible) rebuilds from website for last changes in the release notes
+
+= Announce =
+send out release mails&post on social media #676
+(Personal notice: see also NOTIFY)
+
+
+= TODO =
+- Docker container with sudo rights so that the built deb-packages can be installed and tested in the release pipeline
+- Move ~e/scripts/dev/update-infos-status into release pipeline
+- Move updating of alpine linux docker image into release pipeline
+- "Increment Version Number" could be automized by using parameterized build jobs
```

Comment:
Don't we need the parameter for the dry run? Is it possible to have multiple parameters?

But I think it is fine to have this version increment manually, actually we could do it directly after releases.

Reply:
> Don't we need the parameter for the dry run? Is it possible to have multiple parameters?

Yes, but it possible to have multiple parameters.

> ...we could do it directly after releases

Incrementing the version number needs to be done before the release, at least the one in `CMakeLists.txt`.

Reply:
I meant that we **now** increase to 0.9.4, even if the next release might be in a few months. (So directly after 0.9.x was released, we already pump to 0.9.x+1.)

But this is actually not a good idea, as in future we do not know in advance (for sure) if it will be 1.x+1.0 or 1.x.y+1.

Reply:
Not everything can be incremented at once. The Alpine image for instance depends on the next release tarball already being published. So this is basically done immediately after release / after the tarball upload.